src/tools/singlejar/input_jar_empty_jar_test.cc

```diff
@@ -36,17 +38,14 @@
 #include "src/tools/singlejar/test_util.h"
 #include "googletest/include/gtest/gtest.h"
 
+using bazel::tools::cpp::runfiles::Runfiles;
```

Comment:
You also need to `#include "tools/cpp/runfiles/runfiles.h"` and depend on `@bazel_tools//tools/cpp/runfiles`.
See https://github.com/bazelbuild/bazel/blob/master/tools/cpp/runfiles/runfiles_src.h

Reply:
`tools/cpp/runfiles/runfiles_src.h` is transitively included by `src/tools/singlejar/test_util.h`. Relying on transitive includes is usually bad, but since these are tests and `test_util.h` is kind of a common library for all singlejar tests, I think it is fine.

Reply:
Also, if BuildKite already uses a Bazel version that has `Runfiles::CreateForTest`, I can change all `runfiles_src.h` to `runfiles.h`.